Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "glyph-names"
 version = "0.1.0"
 authors = ["YesLogic Pty. Ltd. <info@yeslogic.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
I would like this change and the `format!` shorthand to be reverted. It pushes up the minimum supported Rust version without providing any clear advantage to the codebase. There are some places where old versions of the compiler might still be in use (E.g. Debian stable) and we may as well remain compatible while there is little downside to the code.

Reply:
Fair enough.